sig-etcd/charter.md

```diff
@@ -0,0 +1,68 @@
+# SIG etcd Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Scope
+
+Owns the etcd project and how it is used by Kubernetes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Development of [etcd] and other repositories under [etcd-io organization]
+- Maintenance of [etcd image] packaged with Kubernetes
+
+[etcd]: https://github.com/etcd-io/etcd
+[etcd-io organization]: https://github.com/etcd-io
+[etcd image]: https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd
+
+#### Cross-cutting and Externally Facing Processes
+
+- Specifying, testing and improving [The Implicit Kubernetes-ETCD Contract]
+- Release process of etcd and other binaries belonging to [etcd-io organization]
+
+[The Implicit Kubernetes-ETCD Contract]: https://docs.google.com/document/d/1NUZDiJeiIH5vo_FMaTWf0JtrQKCx0kpEaIIuPoj9P6A/edit?usp=sharing
+
+### Out of scope
+
+- Structure of data stored in etcd is owned by SIG API Machinery
+
+## Roles and Organization Management
+
+This SIG follows adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Tech Leads
+
+- Release of etcd and other binaries belonging to [etcd-io organization]
+
+### Deviations from [sig-governance]
+
+- SIG-Etcd's participation in the Kubernetes release cycle is limited by Etcd having a different schedule for its releases.  Particularly, Etcd may or may not participate in the Enhancements process depending on whether an upcoming Kubernetes release targets specific new Etcd features.
+- SIG etcd communication utilizes pre-existing forums for communication:
+  - Email: [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev).
+  - Slack: [#etcd](https://kubernetes.slack.com/messages/C3HD8ARJ5/details/) channel on Kubernetes.
+- SIG etcd contributing instructions ([CONTRIBUTING.md]) be defined in etcd project.
+- KEPs created by SIG etcd will not follow Kubernetes release cycle and freezes due.
```

Comment:
FWIW - on the KEP stuff, we do have a good amount of prior art of KEPs that don't align with the release cycle. If they DO align with it, or are involved with it - thats really what the language in the sig-governance is about (we want the sigs to be responsive)

It might be worth steering to make that clear in the governance docs.

Reply:
Removed.